test/unit/utils-content.html

```diff
@@ -88,10 +93,30 @@
         var nodes = elt3.getContentChildren('[dne]');
         assert.equal(nodes.length, 0, 'should find 0 nodes');
       });
-      
+
     });
-  
+
+    suite('isShadowDomChild utils', function() {
+      var elt1 = document.querySelector('#elt1');
+      var elt4 = document.querySelector('#elt4');
+
+      test('isShadowDomChild is false for light children', function() {
+        var span = elt4.querySelector('span');
```

Comment:
Technically, should we be using `Polymer.dom` to search for these element references?